app/components/form/fields/TextField.tsx

```diff
@@ -129,12 +128,17 @@ export const TextFieldInner = <
         title={label}
         type={type}
         error={!!error}
-        aria-labelledby={cn(`${id}-label`, !!tooltipText && `${id}-help-text`)}
-        aria-describedby={tooltipText ? `${id}-label-tip` : undefined}
+        aria-labelledby={`${id}-label ${id}-help-text`}
+        aria-describedby={tooltipText ? `${id}-tooltipText` : undefined}
         onChange={(e) => onChange(transform ? transform(e.target.value) : e.target.value)}
         {...fieldRest}
         {...props}
       />
+      {tooltipText && (
```

Comment:
I want to revisit this one. It might not work with the deconstructed form, but it could make sense to pass the `id` to the tooltip itself, rather than duplicating the tooltipText in the DOM.

Reply:
I have revisited, and chatted with Crespo. It would require a fair bit of prop drilling to get the `id` passed to the appropriate spot, and the `tooltipText` happens infrequently enough that we're okay with an occasional bit of duplication.